spec/views/prosecution_cases/show.html.haml_spec.rb

```diff
@@ -1,23 +1,28 @@
 # frozen_string_literal: true
 
-RSpec.describe 'prosecution_cases/show', type: :view do
-  subject(:render_view) { render }
+RSpec.describe 'prosecution_cases/cd_api/_show_v2.html.haml', type: :view do
```

Comment:
as suggested before, we could get rid off `cd_api/_show_v2.html.haml`,
and maybe the relevant spec.